### DISCUSSION TITLE
Is a menuitem an "user interface component" (or only the menu or menubar) (SC 3.2.1)

### DISCUSSION THREAD

**Discussion by JAWS-test** [w3c/wcag#2646] [Q&A] [answered]
3.2.1:

> When any [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components) receives focus, it does not initiate a [change of context](https://www.w3.org/TR/WCAG21/#dfn-change-of-context).

Is a menuitem an "user interface component" (or only the menu or menubar) (SC 3.2.1).

So is a change of context only forbidden when the menu receives the focus or also forbidden when navigating through the menu with the arrow keys (to the individual menu items)?

Similarly: Is the toolbar the user interface component or also the buttons in the toolbar. Or: Only the grid or also the gridcells and the controls inside the gridcell? Is the tablist the interface component or also the tab inside the tablist?

**Comment by patrickhlauke** [comment, marked as answer]
individual menu items are, i'd say, individual user interface components as well. i'd certainly fail a site that used a menu, and as you cursor through the menu items, it yanks you somewhere else
